3.0/README.md

```diff
@@ -154,6 +154,55 @@ An array of tile endpoints. {z}, {x} and {y}, if present, are replaced with the
 ```
 
 ## 3.15 `vector_layers`
+
+REQUIRED if describing vector tiles.
+
+A JSON object whose value is an array of JSON objects. Each of those JSON objects describes one layer of vector tile data, and MUST contain the following key-value pairs:
+
+* `id` (string): The layer ID, which is referred to as the `name` of the layer in the [Mapbox Vector Tile spec](https://github.com/mapbox/vector-tile-spec/tree/master/2.1#41-layers).
+* `fields` (object): A JSON object whose keys and values are the names and descriptions of attributes available in this layer. Each value (description) MUST be a string. The values MAY describe the underlying data type, such as `String`, `Number`, or `Boolean`. Attributes whose type varies between features MAY be listed as `"String"`.
+
+Each layer object MAY also contain the following key-value pair:
+
+* `description` (string): A human-readable description of the layer's contents.
+* `minzoom` (number): The lowest zoom level whose tiles this layer appears in. MUST be greater than or equal to the tileset's `minzoom`.
+* `maxzoom` (number): The highest zoom level whose tiles this layer appears in. MUST be less than or equal to the tileset's `maxzoom`.
+
+These keys are used to describe the situation where different sets of vector layers appear in different zoom levels of the same tileset, for example in a case where a "minor roads" layer is only present at high zoom levels.
+
+Implementations MUST treat unknown key-value pairs as if they weren't present. However, implementations MUST expose unknown key/values in their API so that API users can optionally handle these keys.
```

Comment:
@GretaCB I've noticed some implementations (mapbox streets tilejson for example) include other keys here. I'm thinking we can document that this is acceptable with the same language from the Structure section https://github.com/mapbox/tilejson-spec/blob/220b1d221a53809fe9d955c611db11d219e99911/3.0/README.md#3-structure. Duplicating the text feels a little redundant, so I'm wondering if we could update the Structure introduction to say something like:

> Implementations MUST treat unknown keys as if they weren't present. **This is true at any depth of the JSON**. However, implementations MUST expose unknown key/values in their API so that API users can optionally handle these keys.

Reply:
I don't think this section is needed at all, given the text elsewhere in the spec.

Reply:
@mapsam what if we link to the Structure section to avoid needing to update text in both places, but also keep a reference in both places to avoid it being overlooked?

Reply:
Sounds good to me @GretaCB - will work an update!